dub.sdl

```diff
@@ -22,13 +22,13 @@ configuration "library" {
 }
 
 configuration "library-nonet" {
-	dependency "vibe-d:http" version=">=0.7.30 <=0.9.0" optional=true
+	dependency "vibe-d:http" version=">=0.7.31 <=0.9.0" optional=true
```

Comment:
It is necessary for 2.074.0 and higher due to https://issues.dlang.org/show_bug.cgi?id=17131
I am perfectly fine with just editing the `dub.selections.json`, but it seemed to me that these two files were kept in sync?